packages/core/src/hooks/useBlockMeta.ts

```diff
@@ -14,13 +15,31 @@ const GET_CURRENT_BLOCK_DIFFICULTY_CALL = MultiCallABI.encodeFunctionData('getCu
  */
 export function useBlockMeta(queryParams: QueryParams = {}) {
   const chainId = useChainId({ queryParams })
+  const { refresh: configRefresh } = useConfig()
```

Comment:
Hmm can you help me understand why `useBlockMeta` is the only one hook that had to be changed in this PR?